.github/workflows/test-af-features.yml

```diff
@@ -30,7 +29,7 @@ jobs:
       - name: App install, test
         run: |
           npm ci
```

Comment:
Näissä voi ehkä muuten pudottaa tän stepin pois, koska test:ci määrittelyssä appsi buildataan eka (dependsOn, turbo.json conffeja, https://turbo.build/repo/docs/reference/configuration#pipeline)

Reply:
Taitaa sitä tarvita kun siinä asennellaan se turbo itsessään, mut ei tarvii niihin apps-kansioihin mennä erikseen asenteleen, which is nice.